app/api/predictions/[id]/route.js

```diff
@@ -5,6 +5,12 @@ const replicate = new Replicate({
   auth: process.env.REPLICATE_API_TOKEN,
 });
 
+// Prevent Next.js / Vercel from caching responses
+// See https://github.com/replicate/replicate-javascript/issues/136#issuecomment-1728053102
+replicate.fetch = (url, options) => {
+  return fetch(url, { ...options, cache: "no-store" });
```

Comment:
Should we allow individual calls to set `cache` is they want?

```suggestion
  return fetch(url, { cache: "no-store", ...options });
```